javascript/operators/import_meta_resolve.json

```
@@ -0,0 +1,50 @@
+{
+  "javascript": {
+    "operators": {
+      "import_meta": {
```

This should be a subfield under `import_meta`, like all other API properties

---

Sorry, I meant to name this 
```suggestion
      "import_meta_resolve": {
```

Should it be something else, though?

---

Yes—you don't need to create a new file. Just add a new sub feature under `import_meta.json`.